src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -982,8 +982,7 @@ private bool IsAuthenticationError(Exception ex)
 
         private void UpdateCacheExpirationTime(KeyValueWatcher changeWatcher)
         {
-            TimeSpan cacheExpirationTime = changeWatcher.CacheExpirationInterval;
-            changeWatcher.CacheExpires = DateTimeOffset.UtcNow.Add(cacheExpirationTime);
+            changeWatcher.CacheExpires = DateTimeOffset.UtcNow.Add(changeWatcher.RefreshInterval);
```

Comment:
The name of `KeyValueWatcher.CacheExpires` remains as is. Just want to make sure if it is as expected.

Reply:
Yea I was trying to think if it should be renamed too, but wasn't sure what made sense and CacheExpires is still accurate for our use I guess. If there's a good way to rename it while referring to refresh I think we could do that though.

Reply:
How about `KeyValueWatcher.NextRefreshTime`?

Reply:
Sounds good to me